transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -305,7 +307,24 @@ static int init_sockaddr(JNIEnv* env, jbyteArray address, jint scopeId, jint jpo
     return 0;
 }
 
-static int socket_type() {
+
+static int getSystemPropertyBoolean(JNIEnv* env, const char* name) {
+    jstring propertyNameString = (*env)->NewStringUTF(env, name);
+    jstring propertyString = (*env)->CallStaticObjectMethod(env, systemClass, systemClassGetPropertyMethodId, propertyNameString);
+    if (propertyString == NULL) {
+        return JNI_FALSE;
+    }
+    const char* property = (*env)->GetStringUTFChars(env, propertyString, 0);
+    int result = strcmp(property, "true" );
+    (*env)->ReleaseStringUTFChars(env, propertyString, property);
+    return result;
```

Comment:
There is actually no way to pass anything to JNI_load which is the method that needs to use it. So we need to do this in JNI